### ISSUE TITLE
倒入数据后数据或者索引分布不均匀

### ISSUE THREAD

**Issue by vperson** [pingcap/tidb#8137]
使用loader工具倒入数据后，使用./table-regions.py工具查看了若干张大表的数据分布和索引分布，出现了很多分布不均匀的问题，查看了官方文档，对于tikv-ctl没找到对应的迁移数据分布的功能。
对于分布不均的数据和索引还有其他解决方法吗？

**Comment by ciscoxll** [comment]
@csuzhangxc PTAL

**Comment by ciscoxll** [comment]
@vperson Thanks for your feedback, You can refer to the [docs here](https://pingcap.com/docs-cn/FAQ/#6-1-2-%E5%A6%82%E4%BD%95%E6%89%93%E6%95%A3%E7%83%AD%E7%82%B9).

**Comment by vperson** [comment]
@ciscoxll 非常感谢的解答，但是我查看了这个参数，这个参数对写入的热点有所缓解。我还是想问下有没有可以迁移region的工具。

**Comment by ciscoxll** [comment]
@vperson Have an http interface, you can use.
`curl http://{TiDBIP}:10080/tables/{db}/{table}/scatter`
`curl http://{TiDBIP}:10080/tables/{db}/{table}/stop-scatter`

**Comment by shenli** [comment]
@vperson Please use 2.1.0-rc.4 or the latest master.

**Comment by nolouch** [comment]
@vperson Yes, the version 2.0.8 not support this future. you can try other schedulers like [`shuffle-leader-scheduler` ](https://github.com/pingcap/docs-cn/blob/master/tools/pd-control.md#scheduler-show--add--remove). because PD do not know anything about table, now we use some API to balance a table.
